app/policies/ticket_policy.rb

```diff
@@ -0,0 +1,32 @@
+class ReservationPolicy < ApplicationPolicy
+  def index?
+    true
+  end
+
+  def show?
+    record.user_id == user.id || user.admin?
+  end
+
+  def update?
+    user.admin?
```

Comment:
I can delete my own reservation but I cannot update it? Just asking :) I want to know your reasoning :)